volumina/widgets/dataExportOptionsDlg.py

```diff
@@ -425,7 +426,7 @@ def validate(self, userAxes, pos):
             # If missing non-singleton axes, maybe intermediate entry
             # (It's okay to omit singleton axes)
             for key in inputSet - userSet:
-                if taggedShape[key] != 1:
+                if taggedShape.get(key, 1) != 1:
```

Comment:
I think this for-loop used to mean "For every input axis not mentioned by the user, make sure that it's a singleton". This change with the `.get[key, 1]` instead of `[key]` doesn't affect that, because `key` is always in `taggedShape` anyway.

If the user entered some axes that are not present in `inputSet` (and we have the necessary magic to interpret that as singletons in the export logic), then there's no further checking we need to do after validating that `userSet.issubset(self.__valid_axes)` and that there are no repeated axes